cli/src/cmd/install.rs

```diff
@@ -22,38 +23,46 @@ impl Cmd for InstallArgs {
     type Output = ();
 
     fn run(self) -> eyre::Result<Self::Output> {
-        install(find_project_root_path()?, self.dependencies, self.opts)
+        install(self.dependencies, self.opts)
     }
 }
 
-#[derive(Debug, Clone, Copy, Default, Parser)]
+#[derive(Debug, Clone, Default, Parser)]
 pub struct DependencyInstallOpts {
     #[clap(help = "install without creating a submodule repository", long)]
     pub no_git: bool,
     #[clap(help = "do not create a commit", long)]
     pub no_commit: bool,
     #[clap(help = "do not print messages", short, long)]
     pub quiet: bool,
+    #[clap(
+        help = "the project's root path. By default, this is the root directory of the current Git repository or the current working directory if it is not part of a Git repository",
+        long,
+        value_hint = ValueHint::DirPath
+    )]
+    pub root: Option<PathBuf>,
 }
 
 /// Installs all dependencies
 pub(crate) fn install(
-    root: impl AsRef<Path>,
     dependencies: Vec<Dependency>,
     opts: DependencyInstallOpts,
 ) -> eyre::Result<()> {
-    let root = root.as_ref();
+    let DependencyInstallOpts { no_git, no_commit, quiet, .. } = opts;
+    let root = match opts.root {
+        Some(r) => r.clone(),
+        None => find_project_root_path()?,
+    };
```

Comment:
rewrite as `let root = if let Some() ..{} else {};`

Reply:
I rewrote it a bit, much smaller footprint and uses `unwrap` like I saw being used in `init.rs`